res/css/_common.scss

```diff
@@ -692,3 +692,15 @@ input[type=text]:focus, input[type=password]:focus, textarea:focus {
         border-radius: $radius;
     }
 }
+
+@define-mixin unreal-focus {
+    outline-width: 2px;
+    outline-style: solid;
+    outline-color: Highlight;
```

Comment:
this will be inconsistent with focus outlines throughout the rest of the app which uses the default browser ones, so we either need to change them everywhere or not introduce inconsistent ones.

Reply:
Is Highlight not the default focus outline colour?

Reply:
It might be the default but lots of a11y users have browser extensions which change the default color/width/style to match their disability and the above does not seem like it'll follow their settings

Reply:
Is there anyone we can refer to who could point us in the right direction?

Reply:
This won't since the focused element is not visible.

Reply:
The style matches on chrome, ff and safari

Chrome:
![image](https://user-images.githubusercontent.com/23084468/89300560-50dbad80-d660-11ea-9446-2dc9e27055e4.png)
![image](https://user-images.githubusercontent.com/23084468/89302736-1e7f7f80-d663-11ea-93e9-ad7ed9ab6163.png)
![image](https://user-images.githubusercontent.com/23084468/89300566-52a57100-d660-11ea-83a5-31b93b582fa5.png)

FF:
![image](https://user-images.githubusercontent.com/23084468/89302475-c8124100-d662-11ea-82cf-9290d9a77461.png)
![image](https://user-images.githubusercontent.com/23084468/89302560-e6783c80-d662-11ea-83c9-6618f3d81633.png)
FF styles seem inconsistent overall:
![image](https://user-images.githubusercontent.com/23084468/89302621-f6901c00-d662-11ea-99c5-6e0316589851.png)


Safari:
![image](https://user-images.githubusercontent.com/23084468/89300612-64871400-d660-11ea-965a-ffc9d1266c79.png)
![image](https://user-images.githubusercontent.com/23084468/89302843-4969d380-d663-11ea-804e-8328fa17fb1d.png)
![image](https://user-images.githubusercontent.com/23084468/89300631-694bc800-d660-11ea-9ead-f8447701a5a9.png)
(I'll fix the left padding issue here)

Reply:
> FF styles seem inconsistent overall:

I believe it is a form element vs non-form element distinction which we should open an issue to track but this is close enough for dogfooding.